interpreters/bas/bas_value.c

```diff
@@ -2026,9 +2042,21 @@ struct String *Value_toWrite(struct Value *this, struct String *s)
 
 struct Value *Value_nullValue(enum ValueType type)
 {
-  static struct Value integer = { V_INTEGER };
-  static struct Value real = { V_REAL };
-  static struct Value string = { V_STRING };
+  static struct Value integer =
+  {
+    V_INTEGER
+  };
+
+  static struct Value real =
+  {
+    V_REAL
+  };
+
+  static struct Value string =
+  {
+    V_STRING
+  };
+
   static char n[] = "";
```

Comment:
Can we remove `n` and use `""` directly?

Reply:
Done.